paperoni/papers2.py

```diff
@@ -38,7 +38,8 @@ class Topic:
 class Author:
     links: Sequence[Link] = ()
     name: str = None
-    affiliations: Sequence[str] = ()
+    affiliations: Sequence[str] = ("",)
```

Comment:
Why?

Reply:
Removed. I needed to iterate over `author.affiliations` to fill table `paper_author` (to link paper to authors), but, when `affiliations` was empty, then the table was not filled. I fixed it by force populate the table even if no affiliation is available.